src/main/java/com/jfrog/ide/idea/ui/webview/model/Cve.java

```diff
@@ -0,0 +1,64 @@
+package com.jfrog.ide.idea.ui.webview.model;
+
+import java.io.Serializable;
+
+public class Cve implements Serializable {
```

Comment:
No serialization happen
```suggestion
public class Cve {
```